contrib/kryo-serializer/src/java/com/twitter/heron/serializer/kryo/KryoSerializer.java

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.twitter.heron.api.serializer;
+package com.twitter.heron.serializer.kryo;
```

Comment:
I think this naming is ok as is. It's consistent with how we name scheduler implementations (i.e., `c.t.h.scheduler.[aurora|marathon|yarn]`). The location of the code could move in or our of the contrib folder without requiring a package renaming.